service/matching/taskQueueManager.go

```diff
@@ -52,13 +52,18 @@ import (
 	"go.temporal.io/server/common/persistence"
 )
 
+type fatalSignalBehavior int
+
 const (
 	// Time budget for empty task to propagate through the function stack and be returned to
 	// pollForActivityTask or pollForWorkflowTask handler.
 	returnEmptyTaskTimeBudget = time.Second
 
 	// Fake Task ID to wrap a task for syncmatch
 	syncMatchTaskId = -137
+
+	emitFatalSignal fatalSignalBehavior = iota + 1
+	suppressFatalSignal
```

Comment:
can we put this in a dedicated const group?

```go
const (
    emitFatalSignal fatalSignalBehavior = iota + 1
    suppressFatalSignal
)
```

Reply:
Sure.